### PR TITLE
COG-1425 Dashboard Tiles Count Query Implementation

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-1425

#### Brief feature description
- A new dora endpoint is created to just return the count of users. This PR implements just the rails portion of that implementation.

### CODE REVIEW

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

File: lib/users/user_repository.rb [hunk 1]
```diff
@@ -81,6 +81,12 @@ def self.search(query, auth_header)
       response.body
     end
 
+    def self.count(query, auth_header)
```
Comment (kelfink) [inline lib/users/user_repository.rb]: I haven't researched the performance benefits of the newer Dora endpoint, but, while this count code looks good, I think it's worth noting that  the functionality of both this PR and the one which created the Dora _count endpoint could instead have been implemented by passing a size=0. parameter to the original search query.   "https://www.elastic.co/guide/en/elasticsearch/reference/current/returning-only-agg-results.html

Comment (Nasir2712) [reply lib/users/user_repository.rb]: I mean still, research needs to be done on how to combine two terms and returning count.

Comment (kelfink) [reply lib/users/user_repository.rb]: Agreed.  after our discussion, I think a good follow-up story would be to optimize this implementation by altering the index with 'copy_to' on the fields we want to aggregate ("group by") and perform one query against the index.

**Pull request merged**